projects/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;netstandard2.0</TargetFrameworks>
```

Comment:
Do you really want to lock out all .NET Framework users? .NET Framework continues to be supported. For us, this would be catastrophic honestly // cc @bording 

To be able to benefit from platform-specific APIs (in case you want to remove .NET Standard 2.0) why not go with for example

<TargetFrameworks>net472;net6</TargetFrameworks>

Reply:
I just need a suggestion for what should be supported and has an acceptable LTS plan. Thanks!

Reply:
The problem is: I don't know how many users of the client would actually depend on [the theoretical targets that is supported by Netstandard2.0](https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0). For NServiceBus we made the tradeoff between ability to target platform-specific APIs while still supporting the users on the .NET Framework by dual targetting and using the .NET Framework version that properly supports Netstandard 2.0 which is 4.7.2

Reply:
Great! Those seem like reasonable choices.

https://docs.microsoft.com/en-us/lifecycle/products/microsoft-net-framework

https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-0